sdk/python/kfp/dsl/_container_op.py

```diff
@@ -745,6 +746,17 @@ def add_volume(self, volume):
         self.volumes.append(volume)
         return self
 
+    def add_toleration(self, tolerations):
+        """Add K8s tolerations
+
+        Args:
+          volume: Kubernetes toleration
+          For detailed spec, check volume definition
```

Comment:
You probably need to rename volume to tolerations

Reply:
done